pages/services/cassandra/2.6.0-3.11.4/configuration/index.md

```diff
@@ -281,3 +281,41 @@ deploy (IN_PROGRESS)
 ### Test your multi-datacenter configuration
 
 Make sure to test your deployment using a {{ model.techShortName }} client.
+
+## Using Volume Profiles
+
+Volume profiles are used to classify volumes. For example, users can group SSDs into a “fast” profile and group HDDs into a “slow” profile. **Note:** Volume profiles are immutable and therefore cannot contain references to specific devices, nodes or other ephemeral identifiers.
+
+DC/OS Storage Service (DSS) is a service that manages volumes, volume profiles, volume providers, and storage devices in a DC/OS cluster.
+
+If you want to deploy Cassandra with DSS, please follow the tutorial here: https://docs.mesosphere.com/services/storage/1.0.0/tutorials/cassandra-dss-volumes/
```

Comment:
Let's push a fix to the error in [their documentation](https://docs.mesosphere.com/services/storage/1.0.0/tutorials/cassandra-dss-volumes/). Step 7 contains an invalid Cassandra service configuration JSON.

Reply:
Yes @mpereira i am creating PR for changes.

Reply:
yup, a week ago I had created this ticket and assigned to their team to fix it: https://jira.mesosphere.com/browse/DCOS-56418

Reply:
Please wrap links as text and not bare html - for example it would be: .... please follow [this tutorial](/services/storage/1.0.0/tutorials/cassandra-dss-volumes/)
Also, the **Note:** is usually put as a standalone colored purple box with the message text inserted at `text`
<p class="message--note"><strong>NOTE: </strong>text</p>. 

https://wiki.mesosphere.com/display/DOCS/Documentation+Style+Guide

Reply:
+1

Reply:
updated as per @ck4adventure 's suggestion
cc: @samvantran